examples/04.Communication/ReadASCIIString/ReadASCIIString.ino

```diff
@@ -6,9 +6,9 @@
   It parses them into ints, and uses those to fade an RGB LED.
 
   Circuit: Common-Cathode RGB LED wired like so:
-  - red anode: digital pin 3
-  - green anode: digital pin 5
-  - blue anode: digital pin 6
+  - red anode: digital pin 3 through 220 ohm resistor
+  - green anode: digital pin 5 through 220 ohm resistor
```

Comment:
120R for green & blue leds would give a better white balance?

Reply:
The problem is that these values are already specified by the text, schematic, and Fritzing in the tutorial that accompanies this example sketch:
https://www.arduino.cc/en/Tutorial/BuiltInExamples/ReadASCIIString
They must be kept synchronized and I don't have access to the Fritzing source file, so it would mean recreating those images from scratch (probably easy enough, but I have almost zero experience with Fritzing so bound to hit some learning curve during that process).

There is also the idea of making the examples accessible to the owners of the [Arduino Starter Kit](https://store.arduino.cc/genuino-starter-kit), which contains 220 ohm resistors, but not 120.

My feeling is that less than ideal resistors are much better than no resistors at all.

Reply:
Ok, I understand. Agreed, better to have them at 220R than not at all.